### PR TITLE
Fix broken image selection in the Media Library

### LINKED ISSUE #7278: Media library - broken image selection
### Expected behavior
Tapping wherever on an image in the Media Library opens the image detail.

### Actual behavior
When a user taps on the bottom-left corner of an image, the image is selected. User can select more images by tapping on them. However there are no action buttons, so the user can't do any action with the selected images. 

### Steps to reproduce the behavior
1. Open media browser
2. Click on the bottom-left corner of an image

##### Tested on [device], Android [version]
Emulator SDK 27
Pixel 2 SDK 27

Might be related to #7277.

### PR DESCRIPTION
**Pull request by jorgecasariego**
Fixes #7278 

To test:
 1. Open media browser
 2. Tapping wherever on an image in the Media Library opens the image detail. (Make sure to also tap on the bottom-left corner of an image) 

This PR could also solve #7277

### CODE REVIEW

**Comment by jorgecasariego** [comment]
@theck13  you can review this PR? 

This is my first attempt to contributing to the project. Please let me know if I have missed a step or need to change the code

Thanks!

File: WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java [hunk 1]
```diff
@@ -285,6 +285,7 @@ public GridViewHolder(View view) {
 
             videoOverlayContainer = (ViewGroup) view.findViewById(R.id.frame_video_overlay);
             selectionCountContainer = (ViewGroup) view.findViewById(R.id.frame_selection_count);
+            selectionCountContainer.setVisibility(mBrowserType.isPicker() ? View.VISIBLE : View.GONE);
```
Comment (theck13) [inline WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java]: This effectively fixes the issue.  I think we can get the same result by setting the visibility in the `onBindViewHolder` method [here](https://github.com/jorgecasariego/WordPress-Android/blob/568682e2ec147ce450dbd6a894a42f79ae9a42dd/WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java#L188) and [here](https://github.com/jorgecasariego/WordPress-Android/blob/568682e2ec147ce450dbd6a894a42f79ae9a42dd/WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java#L197).  The advantage of that would be to have all the visibility logic in one place.

Comment (jorgecasariego) [reply WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java]: Thanks @theck13  

You're right! It is a lot better to do it all in onBindViewHolder

**Comment by jorgecasariego** [comment]
Awesome! Thanks @theck13

**Pull request merged**